test/test-fatal-error.js

```diff
@@ -31,7 +31,7 @@ if (process.argv[2] === 'child') {
     const options = {pid: child.pid};
     // Node.js currently overwrites the command line on AIX
```

Comment:
Nit: we might want to update the comment so that it mentions that node currently overwrites the command line arguments on SmartOS too.